docs/releases/1.26-NOTES.md

```diff
@@ -6,6 +6,8 @@ This is a document to gather the release notes prior to the release.
 
 # Significant changes
 
+* On AWS, bastions are now fronted by a Network Load Balancer.
+
```

Comment:
Maybe we should make an AWS section for specific changes?

Reply:
Done

Reply:
Thanks. Will try to look at the code during weekend.